cubic-server/Entity.cpp

```diff
@@ -21,6 +22,23 @@ Entity::Entity(std::shared_ptr<Dimension> dim) : _dim(dim)
     _id = currentID.fetch_add(1);
 }
 
+void Entity::attack(float damage, Vector3<double> source)
```

Comment:
What if the entity is an non-damagable item, like a item ?

Reply:
@Thomas-Harit Look at CSMC-161, you need to make player inherit from LivingEntity

Reply:
https://edugi.atlassian.net/jira/software/projects/CSMC/boards/3/backlog?selectedIssue=CSMC-161